addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -2662,6 +2663,49 @@ const actions = {
         }
         return htmlString;
     },
+    _getValidatedPartnerIds(unused, partners) {
```

Comment:
docstring, and should not be a store action because it doesn't interact with the store in any way